blackberggroup/VA-OHIL
# added menu dropdown arrow bug fix

## What type of PR is this? (check all applicable)


## Description
Added bug fix to menu expand and collapse arrows appearing for the About and OAM dropdowns

## Related Tickets & Documents
- Monday Task # 6047842305

## QA Instructions, Screenshots, Recordings

Can use keytabbing as well to check menu links and dropdowns on the OHIL home page and iEX landing page. 
Screenshots attached for reference:
<img width="453" alt="Screenshot 2024-02-15 at 9 57 29 AM" src="https://github.com/blackberggroup/VA-OHIL/assets/137827146/d2b893d2-7e7d-4868-bb0d-24ae47ceeffc">
<img width="367" alt="Screenshot 2024-02-15 at 9 57 32 AM" src="https://github.com/blackberggroup/VA-OHIL/assets/137827146/ca6e0481-3f38-4f86-9a4d-410e22a2a3fe">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/.github/PULL_REQUEST_TEMPLATE/index.html
 - _site/assets-3-5/css/styles.css
